main/test/flix/Test.Stratification.flix

```diff
@@ -103,4 +103,33 @@ namespace Test/Stratification {
         };
         ()
 
+    @test
+    def testStratification12(): Unit =
```

Comment:
What about adding a comment to each test cases that says:

// Stratified - The two rules `a` and `b` cannot be in the same program because XYZ?